engine/dlib/src/dlib/log.cpp

```diff
@@ -72,7 +72,7 @@ struct dmLogServer
 };
 
 static dmLogServer* g_dmLogServer = 0;
-static Severity g_LogLevel = LOG_SEVERITY_USER_DEBUG;
+static LogSeverity g_LogLevel = LOG_SEVERITY_USER_DEBUG;
```

Comment:
These changes are from the other PR:
https://github.com/defold/defold/pull/6804

You should only look at Remotery.c